cmd/tendermint/commands/reindex_event.go

```diff
@@ -223,40 +243,45 @@ func eventReIndex(cmd *cobra.Command, es []indexer.EventSink, bs state.BlockStor
 	return nil
 }
 
-func checkValidHeight(bs state.BlockStore) error {
+type checkValidHeightArgs struct {
```

Comment:
This one would probably be fine to pass directly (I imagine it's unlikely to change):

```go
func checkValidHeight(bs state.BlockStore, startHeight, endHeight int64) error {
```

Reply:
yeah, I think either is fine, but I have a minor bias against having sequences of arguments of the same type that can easily be transposed.